tox.ini

```diff
@@ -15,5 +15,5 @@ skip_install = true
 deps =
 	yamllint
 commands = 
-	yamllint --no-warnings rockcraft.yaml
+	yamllint --no-warnings -d '\{rules: \{line-length: \{max: 120\}\}\}' rockcraft.yaml
```

Comment:
So I don't  have to break the formatter for dpkg query

Reply:
If you want to remove lintian overwrite:

```
declare -a fields
fields+=('${db:Status-Abbrev}')
fields+=('${binary:Package}')
fields+=('${Version}')
fields+=('${source:Package}')
fields+=('${Source:Version}')
printf -v dpkg_ops '%s,' "${fields[@]}"
dpkg-query -W -f "${dpkg_ops}\n"
```